madhav/src/main/java/com/lftechnology/phpjava/main/Runner.java

```diff
@@ -0,0 +1,12 @@
+package main.java.com.lftechnology.phpjava.main;
+
+import main.java.com.lftechnology.phpjava.main.views.Menu;
+
+public class Runner {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a private constructor to hide the implicit public one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.java.lftechnology.com:9000/coding_rules#rule_key=squid%3AS1118)